**Baraa**

**Learning Track 3: Data Science and Machine Learning**\
**Stage 0 (Writing)**

**RNA Seq**

It is based on converting RNA into cDNA then fragmental DNA will be read or sequenced using a high-throughput platform this guides researchers to quantify gene expression levels, identify novel transcripts, and uncover alternative splicing events, all of which are crucial for understanding cancer biology (Wang et al., 2009).

<!--[if gte vol 1]><v:shape type id="_x0000_t75" coordsize="21600,21600"
 o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe" filled="f"
 stroked="f">
 <v:stroke joinstyle="miter"/>
 <v: formulas>
  <v:f eqn="if line drawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixel width"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixel width"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v: formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Picture_x0020_1" o:spid="_x0000_i1025" type="#_x0000_t75"
 alt="A diagram of a sequence of dna&#10;&#10;Description automatically generated"
 style='width:277.9pt;height:285.4pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="file:///C:/Users/asus/AppData/Local/Temp/msohtmlclip1/01/clip_image001.jpg"
  o:title="A diagram of a sequence of dna&#10;&#10;Description automatically generated"/>
</v:shape><![endif]--><!--[if !vml]-->![A diagram of a sequence of dna

Description automatically generated](file:///C:/Users/asus/AppData/Local/Temp/msohtmlclip1/01/clip_image002.jpg)<!--[endif]-->

Figure <!--[if supportFields]><span style='font-size:10.0pt;
line-height:150%;font-family:"Times New Roman",serif;mso-ascii-theme-font:major-bidi;
mso-hansi-theme-font:major-bidi;mso-bidi-theme-font:major-bidi'><span
style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>SEQ Figure \* ARABIC <span style='mso-element:
field-separator'></span></span><![endif]-->1<!--[if supportFields]><span
style='font-size:10.0pt;line-height:150%;font-family:"Times New Roman",serif;
mso-ascii-theme-font:major-bidi;mso-hansi-theme-font:major-bidi;mso-bidi-theme-font:
major-bidi'><span style='mso-element:field-end'></span></span><![endif]-->: RNA Seq (Wang et al., 2009)

One of the benefits of RNA-seq is its ability to detect differential gene expression between tumors and normal tissues. compared with other transcriptomics methods is not limited to detecting transcripts that correspond to existing genomic sequences, has a low RNA concentration need compared to others, and is cost-effective (era et al., 2008; Nagalakshmi et al., 2008)

Cancer cells often exhibit altered gene expression profiles that affect their proliferation, survival, and metastasis e.g. breast cancer scRNA-seq leads to molecular subtypes, such as luminal A, luminal B, HER2-enriched, and triple-negative breast cancer (Karaayvaz et al., 2018; Chung et al., 2017).

 RNA-seq can also provide insights into the tumor microenvironment, including the presence of immune cell populations and their interactions with cancer cells as a study by Memorial Sloan Kettering Cancer Center in breast cancer containing one HER2‐positive tumor, two TNBCs, and five ER‐positive tumors using single-cell RNA Seq found different immune composition in every cell (Azizi et al.,2018) .this help Understanding these interactions is essential for developing immunotherapies, a rapidly growing area in cancer treatment

On the other hand, it contributes significantly to personalized medicine by analyzing the transcriptomic profiles of tumors, researchers can identify specific gene expressions that correlate with patient outcomes or responses to treatment (Cheng et al., 2019). Moreover, RNA-seq has facilitated the discovery of novel therapeutic targets.

RNA-seq faces challenges, including data complexity and the need for robust bioinformatics tools to analyze and interpret the data accurately so improvements in sequencing methods and data analysis techniques will likely enhance the resolution and applicability of RNA-seq in research and clinical settings. In summary, RNA sequencing is crucial for advancing cancer research. It helps identify cancer types, discover biomarkers, and find new treatment targets, ultimately improving patient care and outcomes.

****

** **

**Reference**

1- Mateo J, Lord CJ, Serra V, Tutt A, Balmaña J, Castroviejo-Bermejo M, Cruz C, Oaknin A, Kaye SB, de Bono JS. 2019. **A decade of clinical development of PARP inhibitors in perspective**. _Ann Oncol_ **30**: 1437–1447. 10.1093/annonc/mdz192 \[[PMC free article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6771225/)] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31218365)] \[[CrossRef](https://doi.org/10.1093%2Fannonc%2Fmdz192)] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann+Oncol\&title=A+decade+of+clinical+development+of+PARP+inhibitors+in+perspective\&volume=30\&publication_year=2019\&pages=1437-1447\&pmid=31218365\&doi=10.1093/annonc/mdz192&)].

2- Wang Z, Gerstein M, Snyder M. RNA-Seq: a revolutionary tool for transcriptomics. Nat Rev Genet. 2009 Jan;10(1):57-63. doi: 10.1038/nrg2484. PMID: 19015660; PMCID: PMC2949280.

3- Karaayvaz M, Cristea S, Gillespie SM, Patel AP, Mylvaganam R, Luo CC et al. Unravelling subclonal heterogeneity and aggressive disease states in TNBC through single‐cell RNA‐seq. _Nat Commun_. 2018;9(1):3588 10.1038/s41467-018-06052-0. \[[PMC free article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6123496/)] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30181541)] \[[CrossRef](https://doi.org/10.1038%2Fs41467-018-06052-0)] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat+Commun\&title=Unravelling+subclonal+heterogeneity+and+aggressive+disease+states+in+TNBC+through+single%E2%80%90cell+RNA%E2%80%90seq\&volume=9\&issue=1\&publication_year=2018\&pages=3588\&pmid=30181541\&doi=10.1038/s41467-018-06052-0&)].

4- Chung W, Eum HH, Lee HO, Lee KM, Lee HB, Kim KT et al. Single‐cell RNA‐seq enables comprehensive tumour and immune cell profiling in primary breast cancer. _Nat Commun_. 2017;8:15081 10.1038/ncomms15081. \[[PMC free article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5424158/)] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28474673)] \[[CrossRef](https://doi.org/10.1038%2Fncomms15081)] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat+Commun\&title=Single%E2%80%90cell+RNA%E2%80%90seq+enables+comprehensive+tumour+and+immune+cell+profiling+in+primary+breast+cancer\&volume=8\&publication_year=2017\&pages=15081\&pmid=28474673\&doi=10.1038/ncomms15081&)]

5-Cheng Y‐H, Chen Y‐C, Lin E, Brien R, Jung S, Chen Y‐T et al. Hydro‐Seq enables contamination‐free high‐throughput single‐cell RNA‐sequencing for circulating tumor cells. _Nature Communications_. 2019;10(1):2163 10.1038/s41467-019-10122-2. \[[PMC free article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6520360/)] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31092822)] \[[CrossRef](https://doi.org/10.1038%2Fs41467-019-10122-2)] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature+Communications\&title=Hydro%E2%80%90Seq+enables+contamination%E2%80%90free+high%E2%80%90throughput+single%E2%80%90cell+RNA%E2%80%90sequencing+for+circulating+tumor+cells\&volume=10\&issue=1\&publication_year=2019\&pages=2163\&doi=10.1038/s41467-019-10122-2&)].

6- Wang Z, Gerstein M, Snyder M. RNA-Seq: a revolutionary tool for transcriptomics. Nat Rev Genet. 2009 Jan;10(1):57-63. doi: 10.1038/nrg2484. PMID: 19015660; PMCID: PMC2949280.

7- era JC, et al. Rapid transcriptome characterization for a nonmodel organism using 454 pyrosequencing. _Mol. Ecol. _2008;17:1636–1647. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18266620)] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.+Ecol\&title=Rapid+transcriptome+characterization+for+a+nonmodel+organism+using+454+pyrosequencing\&author=JC+Vera\&volume=17\&publication_year=2008\&pages=1636-1647\&pmid=18266620&)] \[[Ref list](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2949280/#R27)].

8- Nagalakshmi U, et al. The transcriptional landscape of the yeast genome defined by RNA sequencing. _Science. _2008;320:1344–1349. \[[PMC free article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2951732/)] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18451266)] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science\&title=The+transcriptional+landscape+of+the+yeast+genome+defined+by+RNA+sequencing\&author=U+Nagalakshmi\&volume=320\&publication_year=2008\&pages=1344-1349\&pmid=18451266&)] \[[Ref list](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2949280/#R18)].

9- Azizi E, Carr AJ, Plitas G, Cornish AE, Konopacki C, Prabhakaran S et al. Single‐Cell Map of Diverse Immune Phenotypes in the Breast Tumor Microenvironment. _Cell_. 2018;174(5):1293‐308.e36. 10.1016/j.cell.2018.05.060. \[[PMC free article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6348010/)] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29961579)] \[[CrossRef](https://doi.org/10.1016%2Fj.cell.2018.05.060)] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell\&title=Single%E2%80%90Cell+Map+of+Diverse+Immune+Phenotypes+in+the+Breast+Tumor+Microenvironment\&volume=174\&issue=5\&publication_year=2018\&pages=1293-308\&pmid=29961579\&doi=10.1016/j.cell.2018.05.060&)]

 

** **

** **

** **
